src/omero_zarr/cli.py

```diff
@@ -196,19 +196,29 @@ def _configure(self, parser: Parser) -> None:
         export.add_argument(
             "--bf",
             action="store_true",
-            help="Use bioformats2raw to export the image.",
+            help="Use bioformats2raw to export the image. Requires"
+            " bioformats2raw 0.3.0 or higher.",
         )
         export.add_argument(
-            "--tile_width", default=None, help="For use with bioformats2raw"
+            "--tile_width",
+            default=None,
+            help="Maximum tile width to read (only for use with bioformats2raw)",
         )
         export.add_argument(
-            "--tile_height", default=None, help="For use with bioformats2raw"
+            "--tile_height",
+            default=None,
+            help="Maximum tile height to read (only for use with bioformats2raw)",
         )
         export.add_argument(
-            "--resolutions", default=None, help="For use with bioformats2raw"
+            "--resolutions",
+            default=None,
+            help="Number of pyramid resolutions to generate"
+            " (only for use with bioformats2raw)",
         )
         export.add_argument(
-            "--max_workers", default=None, help="For use with bioformats2raw"
+            "--max_workers",
```

Comment:
It's not specific to this PR, but I could imagine having a method for setting arbitrary `bioformats2raw` properties:

```
--bf:max_workers=
```

Reply:
interesting. Could even go as far as `--bf2raw-config=<config_file>` with an INI style of YAML list of key/value pairs to be passed to `bioformats2raw`

Reply:
well, if you're going to go that far, I'd take a config file for *all* of the properties here, and separate into `[bioformats2raw]` and `[default]` (or `[omero]`) ;)

Reply:
Should we turn this into an issue?